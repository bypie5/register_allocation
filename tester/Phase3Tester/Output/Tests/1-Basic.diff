--- SelfTestCases/1-Basic.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/1-Basic.out	2020-02-19 16:01:51.000000000 -0800
@@ -1 +1 @@
-6
+Program has no "Main" function.
