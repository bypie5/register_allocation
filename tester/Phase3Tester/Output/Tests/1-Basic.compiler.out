func Main [in 0, out 0, local 0]
$s0 = 4
$s1 = Add($s0 2)
PrintIntS($s1)
ret

